.npmignore

```diff
@@ -10,7 +10,10 @@ babel.config.js
 dist/test
 header.png
 logo.png
+npm-shrinkwrap.json
+package-lock.json
 src
 test_apps
 tsconfig.json
 tslint.json
+yarn.lock
```

Comment:
`yarn.lock` doesn't need to go in the tarball published to the registry, since yarn never takes it into account when installing a package with `yarn add` or as listed in a `package.json` file.